samigo/samigo-app/src/java/com/corejsf/UploadRenderer.java

```diff
@@ -149,7 +149,7 @@ private WrappedUpload wrapUpload(HttpServletRequest request, String name) {
             Part part = request.getPart(name);
             if (part != null) return new WrappedUpload(part);
         } catch (Exception e) {
-            log.error("Failed to get upload part from request. Null will be returned.", e);
+            log.debug("Failed to get upload part from request. Null will be returned.", e);
```

Comment:
log.warn("Failed to get upload part from request, NULL will be returned, {}", e.toString());